app/[locale]/sitemap.xml/route.ts

```diff
@@ -47,6 +48,24 @@ export async function GET(
     return entry;
   });
 
+  const { galleries } = await getSitemapGalleryData(locale);
+  const imageData: any[] = [];
+  galleries.forEach((gallery) => {
+    const { uri, assetAlbum } = gallery;
+
+    assetAlbum.forEach((asset) => {
```

Comment:
issue: this setup is removing all gallery pages (including landing pages) except for image asset pages.

You could map `galleries` to generate the sitemap entries for the gallery landing page, and then maintain a separate array that holds the assets, and then just use the filetype to optionally append the `image:image` addon